dbeam-core/src/test/java/com/spotify/dbeam/jobs/JdbcAvroJobTest.java

```diff
@@ -114,6 +121,17 @@ public void shouldRunJdbcAvroJob() throws IOException {
     List<GenericRecord> records =
         readAvroRecords(new File(DIR, "part-00000-of-00001.avro"), schema);
     Assert.assertEquals(2, records.size());
+
```

Comment:
I believe we can have a separate test for this, e.g. `shouldRunPreCommands`. The idea is to aim for a single behavior per test.

Reply:
it makes sense, I will refactor in order to keep original test separated.

Reply:
@labianchin  PR updated